commands/pass.go

```diff
@@ -0,0 +1,59 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/bndw/pick/utils/pswdgen"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "pass",
+		Short: "Generate a password without storing it",
+		Long:  "The pass command is used to generate a password without storing it.",
+		Run: func(cmd *cobra.Command, args []string) {
+			runCommand(Pass, cmd, args)
+		},
+	}
+	cmd.Flags().Int("length", pswdgen.DefaultPasswordLength, "the length of the generated password")
```

Comment:
And that's my issue. I frequently need to generate passwords and that's something pick should be able to provide natively.

I think making `pass` default to non-interactive is the way forward, but we'll need some way to override it, maybe a boolean `--interactive` flag.